service/local-dev/skaffold.yaml.template

```diff
@@ -1,5 +1,5 @@
 #This is a Skaffold configuration, which lets developers continuously push new images to their development namespaces.
-apiVersion: skaffold/v2alpha4
+apiVersion: skaffold/v2beta17
```

Comment:
Will this require an update of skaffold on our machines?

Reply:
Yes, the locally installed-version of Skaffold should complain if it is not recent enough to support this config version.